pytorch/training/docker/1.11/py3/Dockerfile.cpu

```diff
@@ -296,7 +298,7 @@ RUN pip install --upgrade pip --trusted-host pypi.org --trusted-host files.pytho
     smclarify \
     "sagemaker>=2,<3" \
     sagemaker-experiments==0.* \
-    "sagemaker-pytorch-training>=2.5.1,<3"
```

Comment:
Do we know why this was pinned originally?

Reply:
I'm not sure, but the to consume latest protobuf deprecation fix and the hopper fix, we need to upgrade.

Reply:
Sounds good.